apps/hip/tests/test_views.py

```diff
@@ -319,3 +320,95 @@ def test_djangoadmin_login_redirects_to_cms_and_admin_login_post(db, client, nex
     # persists to the regular login page.
     assert 302 == response.status_code
     assert f"{reverse('login')}?next={next_url}" == response.url
+
+
+def test_get_document_success(db, client):
+    """Sending valid parameters should return the expected document."""
+    document = DocumentFactory()
+    url = reverse(
+        "get_document",
+        kwargs={"document_id": document.id, "document_name": document.filename},
+    )
+    response = client.get(url)
+    assert response.status_code == 200
+    assert (
```

Comment:
This assertion failed for me locally.

```

    def test_get_document_success(db, client):
        """Sending valid parameters should return the expected document."""
        document = DocumentFactory()
        url = reverse(
            "get_document",
            kwargs={"document_id": document.id, "document_name": document.filename},
        )
        response = client.get(url)
        assert response.status_code == 200
>       assert (
            response.headers.get("Content-Disposition")
            == f'inline; filename="{document.filename}"'
        )
E       AttributeError: 'FileResponse' object has no attribute 'headers'
```

Reply:
It's possible that my db is just very outdated.

Reply:
Thanks for taking a look!
That's strange that it says a `FileResponse` wouldn't have a `headers` attribute, since [it's defined in Django's source code](https://github.com/django/django/blob/stable/3.2.x/django/http/response.py#L100) (`FileResponse` inherits from `StreamingHttpResponse`, which inherits from `HttpResponseBase`). Could you check the version of Django that you have installed locally? Maybe it's an outdated version where `FileResponse.headers` is not defined?

Reply:
My virtualenv for philly is on 3.1.8. updating resolved the issue.